pkg/security/ebpf/c/include/constants/offsets/filesystem.h

```diff
@@ -252,6 +252,20 @@ int __attribute__((always_inline)) get_ovl_lower_ino_new(struct dentry *dentry)
     return get_dentry_ino(lower);
 }
 
+int __attribute__((always_inline)) get_ovl_lower_ino_new_2(struct dentry *dentry) {
```

Comment:
what does this func name mean?

Reply:
we had `_old` and `_new` so this is the `_new_new` version.. not sure how to distinguish the version more clearly

Reply:
alright so I renamed all 3 functions with more descriptive names